bin/branding.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+import os
+import string
+import fnmatch
+import sys
+from optparse import OptionParser
+
+path = os.path.dirname(os.path.realpath(__file__))
+branding_values = dict()
+
+# Read branding environment variable list from branding file
+# For each variable read the corresponding env variable and store to branding map
+with open(os.path.join(path, 'branding.list'), 'r') as branding:
+	lines = filter(lambda x: x and not x.startswith('--'), branding.read().splitlines())
+	branding_values = {x:os.environ.get(x) for x in lines}
+
+# Read list of environment variables from file, where each is a path to a file
+# And for each read the content of that file to the branding map
+# brand-file contents from list of file name env variables and store to branding map as well
+with open(os.path.join(path, 'branding-files.list'), 'r') as branding:
+	lines = filter(lambda x: x and not x.startswith('--'), branding.read().splitlines())
```

Comment:
I had been thinking of this too. So far it supports existing formatting, but I'll add it.